components/switch/README.md

```diff
@@ -2,21 +2,31 @@
 
 Switch components are similar to button toggles and checkboxes, except that they have on/off semantics that reflect their appearance more closely. In contrast to checkboxes, changes are generally expected to be immediate as opposed to submitted as part of a form.
 
-## d2l-switch
+<!-- docs: demo name:d2l-switch -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/switch/switch.js';
+  import '@brightspace-ui/core/components/switch/switch-visibility.js';
+</script>
+<d2l-switch text="Switcheroo" tooltip="Click to toggle!"></d2l-switch>
+<d2l-switch-visibility text="Visibility Switch" tooltip="Click to toggle!" on></d2l-switch-visibility>
+```
 
+## Switch [d2l-switch]	
 The `d2l-switch` element is a generic switch with on/off semantics.
 
-![Switch](./screenshots/switch.png?raw=true)
-
+<!-- docs: demo live name:d2l-switch -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/switch/switch.js';
+  import '@brightspace-ui/core/components/switch/switch-visibility.js';
 </script>
-
-<d2l-switch text="Dark Mode" on></d2l-switch>
+<d2l-switch></d2l-switch>
 ```
+<!-- docs: start hidden content -->
+![Switch](./screenshots/switch.png?raw=true)
```

Comment:
I think for consistency we'd want to leave this where it was above the demo and then wrap i the hidden comments there.

Reply:
Okay cool, I was just trying to conglomerate some of the hidden items but makes sense to keep a consistent format